bonita-integration-tests/bonita-integration-tests-client/src/main/java/org/bonitasoft/engine/process/document/DocumentIT.java

```diff
@@ -688,14 +704,17 @@ public void getArchivedVersionOfDocuments() throws BonitaException {
         try {
             final Document beforeUpdate = getAttachmentWithoutItsContent(processInstance);
             final Document doc = BuildTestUtil.buildDocument(beforeUpdate.getName());
-            getProcessAPI().attachNewDocumentVersion(processInstance.getId(), beforeUpdate.getName(), doc.getContentFileName(), doc.getContentMimeType(),
-                    "contentOfTheDoc".getBytes());
+            getProcessAPI().attachNewDocumentVersion(processInstance.getId(), beforeUpdate.getName(), "newContent1.file", doc.getContentMimeType(),
+                    "contentOfTheDoc1".getBytes());
+            getProcessAPI().attachNewDocumentVersion(processInstance.getId(), beforeUpdate.getName(), "newContent2.file", doc.getContentMimeType(),
+                    "contentOfTheDoc2".getBytes());
             getAttachmentWithoutItsContent(processInstance);
```

Comment:
the new test is here: getArchivedVersionOfDocuments

the other changes was an enhancement of the test searchDocuments